internal/templates/data/common/js/cursor/sdk.mdc

```diff
@@ -0,0 +1,95 @@
+---
+description: Agentuity JavaScript SDK API Reference
+globs: "src/agents/**/*.{js,ts}"
+---
+
+# Agentuity JavaScript SDK
+
+The Agentuity JavaScript SDK provides a powerful framework for building AI agents in JavaScript and TypeScript. This cursor rules file helps you navigate the SDK's core interfaces and methods.
+
+## Core Interfaces
+
+### AgentHandler
+
+The main handler function type for an agent:
+
+```typescript
+type AgentHandler = (
+  request: AgentRequest,
+  response: AgentResponse,
+  context: AgentContext
+) => Promise<AgentResponseType>;
+```
+
+### AgentRequest
+
+The `AgentRequest` interface provides methods for accessing request data:
+
+- `request.trigger`: Gets the trigger type of the request
+- `request.data.contentType`: Gets the content type of the request payload
+- `request.metadata(key, defaultValue)`: Gets metadata associated with the request
```

Comment:
move request.metadata to be after request.trigger

add request.get just after request.metadata which is:

```typescript
/**
	 * get the metadata value of the request
	 */
	get(key: string, defaultValue?: Json): Json;
```